menu/MenuItemCard/MenuItemCard.tsx

```diff
@@ -12,34 +12,61 @@ import { opacity } from '@cloudinary/url-gen/actions/adjust';
 interface MenuItemCardProps {
   menuItemData: IMenuItem;
   cdn: Cloudinary;
+  isModal?: boolean;
```

Comment:
Looks like the logic of 2 different components mixed in a single one,  and such boolean is a bright sign of it.  Should we split simple MenuItemCard from MenuItemCardModal?

Reply:
I had that feeling in the end, whereas in the beginning it seemed like a good idea.. :)